### DISCUSSION TITLE
ZFW incorrect on Simbrief import

### DISCUSSION THREAD

**Discussion by Tigerhawk63** [flybywiresim/aircraft#4036] [Help]
When I import the w/b info from simbrief I am getting a different zfw than what was in simbrief.
It appears that the payload is correct, but the dow in the Simbrief flightplan is aprox 43.3 tonnes whereas the empty weight in msfs is showing as a touch over 41 tonnes.
Am I doing something wrong or is this a bug?

**Comment by ExampleWasTaken** [comment]
I would think that this is because SimBrief calculates for an A320ceo and you are using an A320neo.

**Comment by B1900Mech** [comment]
Have you tried creating a new aircraft profile in Simbrief? I think easyjet simpilot has a tutioral on You Tube.

**Reply by Tigerhawk63** [reply]
Yes I have, but for some reason it pulls through the empty weight as 43.3 tonnes even though the empty weight in the Simbrief profile is 41005 to match the MSFS figure.
Strangely enough, on a couple of occasions it has set the DOW as 41005, but I am unable to work out why!
<img width="410" alt="GTTND" src="https://user-images.githubusercontent.com/80625691/113505637-0afd1f00-9538-11eb-9dbc-537d3ca6f196.PNG">

**Reply by Tigerhawk63** [reply]
Further investigation has found some anomalies. It is due to the format of OFP used in Simbrief.  If I use BAW format then the ZFW is pulled through correctly.  However if I use BAW format the Flypad crashes if I try to open the OFP within it.  Using EZY (which I used because I knew it worked from watching "Easyjetsimpilot's" Youtube tutorials result in the ZFW being pulled through incorrectly, but the OFP being displayed in the Pad.
One or the other? On balance, not having to mess with the W & B in MSFS is probably better, after all I can still access the OFP from Simbrief in Chrome.

**Reply by ExampleWasTaken** [reply]
Not all OFP formats are currently working and only LIDO is officially supported at the moment. Could you try with that format again?

**Reply by Tigerhawk63** [reply]
I'll have a look tomorrow, thank you for your assistance

**Reply by Tigerhawk63** [reply]
Thank you, that worked.  Using LIDO format both the ZFW was pulled through correctly, and the OFP opened in the pad.
